ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -145,6 +146,11 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		self.tableView.dragDelegate = self
 		self.tableView.dropDelegate = self
 		self.tableView.dragInteractionEnabled = true
+
+		let actionsBarButton: UIBarButtonItem = UIBarButtonItem(title: "● ● ●", style: .done, target: self, action: #selector(uploadsBarButtonPressed))
+		actionsBarButton.setTitleTextAttributes([.font :UIFont.systemFont(ofSize: 10)], for: .normal)
```

Comment:
I think we should start thinking about moving fonts into Theming support.

Reply:
I think there is no current mechanism implemented to allow fonts with Themes but I'm with you, we should code it and move all the fonts there.

Reply:
@pablocarmu yeah, what do you think of creating a separate issue for this topic?

Reply:
Sure, I'll open it so we can discuss there.